packages/@ourworldindata/grapher/src/stackedCharts/StackedUtils.test.ts

```diff
@@ -26,6 +35,18 @@ it("can add fake points", () => {
     expect(series[1].points[1].position).toEqual(2002)
 })
 
+it("can enforce uniform spacing on the x-axis", () => {
+    expect(seriesArr[1].points[1]).toEqual(undefined)
+    expect(seriesArr[1].points[2]).toEqual(undefined)
+    expect(seriesArr[1].points[3]).toEqual(undefined)
+    const series = withMissingValuesAsZeroes(seriesArr, {
+        enforceUniformSpacing: true,
+    })
+    expect(series[1].points[1].position).toEqual(2001)
+    expect(series[1].points[2].position).toEqual(2002)
+    expect(series[1].points[3].position).toEqual(2003)
+})
+
```

Comment:
I feel it'd also be worthwhile to have some isolated tests of `withUniformSpacing` in here.